rilldata/rill
# TimeDimensionDisplay DST day conversion fix

## Checklist

## Summary
#### Issue addressed: 

https://github.com/rilldata/rill/issues/3560#issuecomment-1881384653
#### Details:


## Steps to Verify

1. Open 311 Operational Metrics dashboard.
1. Set `first_day_of_week: 7`
1. In Chrome Developer Tools -> Sensors modify Location to `America/New_York` Timezone ID.
1. Drill into Median Time to Resolve Ticket (Hours) from main dashboard view
1. Adjust date range to Oct 29 - Nov 30 and make sure that metric trends is set to week
1. Check that the column header in the TDD shows Nov 5 (Sun) as first day of week.
1. Select time subrange ending with Nov 5 by dragging the mouse over the chart.
1. Check the subrange control shows Nov 5.

![image](https://github.com/rilldata/rill/assets/14215045/bb010bcd-b8a3-4cff-91f9-75d15fbb47fc)
